src/sass/components/_gallery.scss

```diff
@@ -0,0 +1,319 @@
+                                
+                                &-btnContainer {
+                                    padding: 5px 0;
+
+                                    &-btn {
+                                        &:hover {
+                                            border: 1px solid $primary;
+                                            background-color: transparent;
+                                            color: $primary;
+                                        }
+                                    }
+                                              
+                                }
+                            }    
+                            
+                            &-dataContainer {
+                                position: absolute;
+                                bottom: 0;
+                                right: 0;
+                                display: flex;
+                                flex-direction: column;  
+
+
+                                &-name {
+                                    position: relative;
+                                    max-width: 400px;
+                                    padding: 6px 15px;
+                                    margin-bottom: 28px;
+                                    box-shadow: 0 3px 6px rgba(1,2,2,0.24);
+                                    background-color: $white;
+                                    
+                                    &::before {
+                                        content: ' ';
+                                        position: absolute;
+                                        border-left: 20px solid transparent;
+                                        border-right: 20px solid transparent;
+                                        transform: rotate(-45deg); 
+                                        left: -14px;
+                                        top: -4px;
+                                        border-bottom: 20px solid #2a2a2a;
+                                    }
+
+                                    &::after {
+                                        content: ' ';
+                                        position: absolute;
+                                        border-left: 20px solid transparent;
+                                        border-right: 20px solid transparent;
+                                        transform: rotate(-45deg); 
+                                        right: -14px;
+                                        bottom: -4px;
+                                        border-top: 20px solid #2a2a2a;
+                                    }
+
+                                    .content {
+                                        text-align: center;
+                                        padding: 20px;
+                                    
+                                        &-h5 {
+                                          color: $primary;
```

Comment:
W projekcie jest inny kolor nagłówka.

Reply:
Chylę czoła, że to wyhaczyłeś :) ja pewnie bym nie zauważyła :) poprawione :)